packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -110,6 +109,28 @@ export default createRule<Options, MessageIds>({
     const checker = services.program.getTypeChecker();
     const compilerOptions = services.program.getCompilerOptions();
 
+    function getCommentDefaultCase(
+      node: TSESTree.SwitchStatement,
+    ): TSESTree.Comment | undefined {
+      const defaultCaseCommentConstants = ['no default', 'No Default'];
```

Comment:
What do you think about using the same pattern as eslint `default-case`? ("i" flag for case-insensitive)

https://github.com/eslint/eslint/blob/35a8858d62cb050fa0b56702e55c94ffaaf6956d/lib/rules/default-case.js#L6-L8
```TS
const DEFAULT_COMMENT_PATTERN = /^no default$/iu;
```